address-watch/src/agent.js

```diff
@@ -1,34 +1,38 @@
 const { Finding, FindingSeverity, FindingType } = require('forta-agent');
 
 // load config file
-const addressList = require('./address-watch.json');
+const config = require('../bot-config.json');
 
-// load configuration data from agent config file
-const { aaveEverestId: AAVE_EVEREST_ID } = require('../agent-config.json');
+// load configuration data from bot config file
+const {
+  developerAbbreviation: developerAbbrev,
+  protocolName,
+  protocolAbbrev,
+  contracts,
+} = config;
```

Comment:
It doesn't look like `config` is used anywhere else, so these can be combined:
```suggestion
// load configuration data from bot config file
const {
  developerAbbreviation: developerAbbrev,
  protocolName,
  protocolAbbrev,
  contracts,
} = require('../bot-config.json');
```